{{cookiecutter.app_name}}/README.md

```diff
@@ -12,12 +12,21 @@
 
 ### Development
 
-- Start development server
+#### Build dependencies
+This project using [`air`](https://github.com/cosmtrek/air) for live reloading. It need to be built as a binary file in `$GOPATH`.
```

Comment:
Please recheck the English syntax:

```suggestion
This project is used [`air`](https://github.com/cosmtrek/air) for live reloading. It needs to be built as a binary file in `$GOPATH`.
```

But I recommend this phrasing:

```suggestion
[`air`](https://github.com/cosmtrek/air) is used for live reloading. It needs to be built as a binary file in `$GOPATH`.
```

Reply:
Updated README in e8fbe849abed486a3a96b1ae6dd1f5c98959a54c